app/core/Encryptor/Encryptor.test.ts

```diff
@@ -294,7 +318,7 @@ describe('Encryptor', () => {
 
       const importedKey = await encryptor.importKey(result.exportedKeyString);
       expect(importedKey).toHaveProperty('exportable', true);
-      expect(importedKey).toHaveProperty('lib', 'original');
+      expect(importedKey).toHaveProperty('lib', ENCRYPTION_LIBRARY.quickCrypto);
```

Comment:
Very nit: But we do use raw literal in all other tests (with `.toHaveProperty`), so to be consistent, we could do the same here, or re-use the `enum` everywhere?

```suggestion
      expect(importedKey).toHaveProperty('lib', 'quick-crypto');
```